docs(azure): add azure baking (#1705)
armory/docs: content/en/continuous-deployment/spinnaker-user-guides/azure-guides/bake-images.md

  * [Baking images](https://spinnaker.io/docs/setup/other_config/bakery/) in Spinnaker
  * [Virtual machines and images in Azure](https://learn.microsoft.com/en-us/azure/virtual-machines/)

* You are deploying to [Azure Portal](https://learn.microsoft.com/en-us/azure/azure-portal/) and have [configured your Azure account](https://spinnaker.io/docs/setup/install/providers/azure/) in Spinnaker.

## Configure your bake using an Azure managed image

1. Add a **Bake** stage in your pipeline.
1. In the **Bake Configuration** section:

   1. Select your account from the **Account** list.
   1. Select a region in the **Regions** list.
   1. Select **Managed Images**.
   1. Select an image from the **Managed Image** list, which contains the images from your Azure account.
      {{< figure src="/images/user-guides/azure/select-managed-image.png" alt="Select a managed image." >}}

   1. Select a value in the **Package Type** field. 

      * If you are using a Linux image, you need to select the distro: 

         * **DEB**: For example, Debian and Ubuntu
         * **RPM**: For example, CentOS and Fedora

>       * If you are using a Windows image, select the `chocolatey` package manager.

Author: The option is not displayed in the ui
```suggestion
      * If you are using a Windows image, it will use `chocolatey`.
```
